packages/netsuite-adapter/test/filters/author_information/saved_searches.test.ts

```diff
@@ -82,6 +83,11 @@ describe('netsuite saved searches author information tests', () => {
     expect(savedSearch.annotations[CORE_ANNOTATIONS.CHANGED_BY] === 'user 1 name').toBeTruthy()
   })
 
+  it('should add last modified date to elements', async () => {
+    await filterCreator(filterOpts).onFetch?.(elements)
+    expect(savedSearch.annotations[CORE_ANNOTATIONS.CHANGED_AT] === '01/28/1995').toBeTruthy()
```

Comment:
This is super nit in this case, but just be aware that `toBeTruthy()` may be a bit confusing so when possible I prefer being more explicit.
```suggestion
    expect(savedSearch.annotations[CORE_ANNOTATIONS.CHANGED_AT]).toEqual('01/28/1995')
```